pypesto/petab/importer.py

```diff
@@ -100,40 +100,29 @@ def __init__(
             Whether to use hierarchical optimization or not, in case the
             underlying PEtab problem has parameters marked for hierarchical
             optimization (non-empty `parameterType` column in the PEtab
-            parameter table). Required for ordinal and nonlinear-monotone data.
-        ordinal:
-            Whether ordinal data is used in the optimization problem. In this
-            case the Optimal Scaling approach will be used to integrate it
-            in an inner optimization subproblem. This requires the `hierarchical`
-            flag to be set to `True`.
-        nonlinear_monotone:
-            Whether nonlinear-monotone data is used in the optimization problem.
-            In this case the spline approximation approach will be used to
-            integrate it in an inner optimization subproblem. This requires the
-            `hierarchical` flag to be set to `True`.
-        inner_solver_options:
-            Options of the inner solver, passed to constructors of inner solvers.
+            parameter table). Required for ordinal, censored and nonlinear-monotone data.
+        inner_options:
+            Options for the inner problems and solvers.
             If not provided, default options will be used.
         """
         self.petab_problem = petab_problem
         self._hierarchical = hierarchical
-        self._ordinal = ordinal
-        self._nonlinear_monotone = nonlinear_monotone
 
-        if self._ordinal and not self._hierarchical:
-            raise ValueError(
-                "Ordinal data requires hierarchical optimization to be "
-                "enabled.",
-            )
-        if self._nonlinear_monotone and not self._hierarchical:
+        self._non_quantitative_data_types = (
```

Comment:
start new line with `\` rather than using parenthesis?

Reply:
Black seems to not like this, reverses it to ().